source/developers/quickstarts/first_openedx_pr.rst

```diff
@@ -303,5 +303,7 @@ changes as is and merge them.
 .. note::
    .. include:: /documentors/how-tos/reusable_content/sign_agreement.txt
 
+If you are now looking for something to work on, please see `How to start contributing to the Open edX code base`_.
```

Comment:
**Author Note:** https://github.com/openedx/docs.openedx.org/blob/main/source/documentors/how-tos/reusable_content/sign_agreement.txt contains this same link, and doesn't supply the reference, so I imagine it is already available. At the same time, it's a little off, and maybe the link should be repeated?

Reply:
I guess this is what people are using, so I'll leave as-is. See links here: https://github.com/openedx/docs.openedx.org/blob/8a58a990575d39345564b765a1af14e8d2b9c90a/source/links.txt#L21